### PR TITLE
Bootstrap Version

### PR DESCRIPTION
**Pull request by shellbell22**
Revised JS and html, looking for reviews of this code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,22 +4,25 @@
 <head>
 	<meta charset="UTF-8">
 	<title>PirateBartender</title>
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 </head>
```
Comment (nicole-lindsay) [inline index.html:9]: Indentation & spacing between elements should be uniform throughout

File: js/app.js [hunk 2]
```diff
@@ -1,5 +1,3 @@
-//create objects
-//[] {}
 //list of all questions
 var Question = {
```
Comment (nicole-lindsay) [inline js/app.js]: variable does not follow naming conventions, should be lowercase

File: js/app.js [hunk 3]
```diff
@@ -18,17 +16,13 @@ var Ingredients = {
...
 };
 
-//not sure why you would need this object in this example...
-var Pantry = {
-};
 
-// original bartender object, not sure what to put here
+// original bartender object
 var Bartender = function(answers) {
   this.answers = [];
```
Comment (nicole-lindsay) [inline js/app.js:21]: instead of `this.answers = [];` 
use `this.answers = answers;`

File: js/app.js [hunk 4]
```diff
@@ -77,13 +65,9 @@ $(function() {
     //create a bartender
     var marioBartender = new Bartender();
 
-
     // the bartender asks the questions (prints them out on screen)
     marioBartender.makeQuestions();
 
-    //var drinkIngredients = new Ingredients();
-
```
Comment (nicole-lindsay) [inline js/app.js:85]: blank space should be uniform throughout

File: js/app.js [hunk 5]
```diff
@@ -38,13 +32,10 @@ Bartender.prototype.makeDrink = function() {
     $('input:checked').each(function(item, input) {
         //push the keys into an array
         var key = $(input).val();
-        console.log(key);
         //this=input
         bartender.answers.push(key);
     });
 
-  //  console.log('keyarray is' + keyarray.length);
```
Comment (nicole-lindsay) [inline js/app.js:46]: remove console.log where it is no longer needed

File: js/app.js [hunk 6]
```diff
@@ -18,17 +16,13 @@ var Ingredients = {
     'fruity': ['slice of orange', 'dash of cassis', 'cherry on top']
 };
 
-//not sure why you would need this object in this example...
```
Comment (nicole-lindsay) [inline js/app.js:21]: ingredients should be within the pantry, possibly
`var pantry = function(ingredients) {
    this.ingredients = ingredients
}`

File: js/app.js [hunk 7]
```diff
@@ -64,9 +54,7 @@ Bartender.prototype.makeDrink = function() {
 Bartender.prototype.makeQuestions = function(questions) {
     //go through the questions and print them for the user
     for (item in Question) {
-        //for(i=0; i< Question.length; i++) {
-        console.log(item, Question[item]);
```
Comment (cgberlin) [inline js/app.js:68]: There seems to be a lot of console logs. I am not sure if these are still necessary, but you may want to remove some of the less important ones just to keep it clean; or change them to give error messages should something go wrong.

File: js/app.js [hunk 8]
```diff
@@ -64,9 +54,7 @@ Bartender.prototype.makeDrink = function() {
...
-        $('#questionsdiv').append(Question[item] + '<input type="checkbox" value="' + item + '"/> <br>');
+        $('#questionsdiv').append('<div class="checkbox"><label><input type="checkbox" value="' + item + '"/>'+ Question[item] + '</label></div>');
     }
```
Comment (cgberlin) [inline js/app.js:56]: You could append both of these things to the #questiondiv at once.

**Comment by shellbell22** [comment]
Thank you @nicole-lindsay and @cgberlin for your comments!

**Pull request merged**